editor/import/resource_importer_texture.cpp

```diff
@@ -433,7 +433,7 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 		image->lock();
 		for (int i = 0; i < height; i++) {
 			for (int j = 0; j < width; j++) {
-				image->set_pixel(i, j, image->get_pixel(i, j).inverted());
+				image->set_pixel(j, i, image->get_pixel(j, i).inverted());
```

Comment:
It's actually more common to use `i` for width (`x` coordinate) and `j` for height (`y` coordinate), which avoids such confusion.

Reply:
Changed.